platforms/core-configuration/configuration-problems-base/src/main/kotlin/org/gradle/internal/configuration/problems/PropertyProblem.kt

```diff
@@ -61,13 +61,15 @@ typealias StructuredMessageBuilder = StructuredMessage.Builder.() -> Unit
 
 data class StructuredMessage(val fragments: List<Fragment>) {
 
-    override fun toString(): String = fragments.joinToString(separator = "") { fragment ->
+    fun asString(quote: Char = '\'') = fragments.joinToString(separator = "") { fragment ->
```

Comment:
💅  The `asString` is too similar to `toString`. What about naming it more explicitly, something like `format()` or `render()` ?

Reply:
Agreed, since this takes a parameter, it is more than a variant of `toString()`